backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.crm.model.db;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name = "cars")
+@Data
```

Comment:
Don't use @Data annotation in Entity. We rather prefer generate our own equals, hashCode and toString methods. Instead of @Data please use @Getter and @Setter

Reply:
done